test/unit/carousel.ts

```diff
@@ -0,0 +1,698 @@
+import { utils } from '@storefront/core';
+import * as sinon from 'sinon';
+import Carousel, { isSwipeToNext } from '../../src/carousel';
+import suite from './_suite';
+
+suite('Carousel', ({ expect, spy, stub }) => {
+  let carousel: Carousel;
+  let DEFAULT_SETTINGS;
```

Comment:
Why is this in a let? As far as I can tell it only gets used twice, being set to the same thing.